tklinenums/tklinenums.py

```diff
@@ -12,14 +12,16 @@
 def scroll_fix(delta: int) -> int:
     """Corrects scrolling numbers across platforms"""
 
-    # The scroll events passed by MacOS are different from Windows and Linux
+    # The scroll events passed by macOS are different from Windows and Linux
     # so it must to be rectified to work properly when dealing with the events.
     # Originally found here: https://stackoverflow.com/a/17457843/17053202
-    if delta in (4, 5):
-        # Linux device
-        result = delta / 4 if delta == 4 else -delta / 5
-    result = -delta if SYSTEM == "Darwin" else delta / 120
-    return int(result)
+    if delta in (4, 5):  # X11 (maybe macOS with X11 too)
+        return 1 if delta == 4 else -1
+    elif SYSTEM == "Darwin":  # macOS
```

Comment:
No elif. Guard clauses just make it need a regular if.

Reply:
Okay

Reply:
Why?

Reply:
I mean, it does the same thing, but what we're doing is basically an `elif`, so it would make sense to me.

Reply:
I personally really hate the look of if elif statements where it's unnecessary and I like guard clause syntax which aims to eliminate them.